benchmark.go

```diff
@@ -132,6 +137,9 @@ func runBenchmark(out output, allOpts Options, m benchmarkMethod) {
 		states[i] = newBenchmarkState(statter)
 	}
 
+	progressBar := pb.StartNew(opts.MaxRequests)
```

Comment:
Thanks for this. Found a race testing this with Duration. After playing around with the library I forked it and might try to upstream more fixes. Seems small enough and active that this can wait for the upstream to take my fix or we can use my fork and improve the library for our needs as well. 

https://github.com/cheggaaa/pb/pull/103